ResourceSchemas/OrganizationSettings.Schema.json

```diff
@@ -0,0 +1,347 @@
+            "category": "Integrations",
+            "isAdvanced": true,
+            "isSecret": false,
+            "readOnly": false
+        },
+        "LogAnalyticsSkipUnchanged": {
+            "type": "string",
+            "title": "Log Analytics Skip Unchanged",
+            "description": "Option to only send changed data to log analytics.",
+            "defaultTextIfEmpty": "Reporting is not yet installed. Follow <a href=\"https://simeoncloud.github.io/docs/#/reporting\">this guide</a> or complete these variables to install reporting.",
+            "category": "Integrations",
+            "enum": [
+                "true",
+                "false"
+            ],
+            "isAdvanced": true,
+            "isSecret": false,
+            "readOnly": false
+        },
+        "LogAnalyticsTenant": {
+            "type": "string",
+            "title": "Log Analytics Tenant",
+            "description": "The tenant in which log analytics is installed.",
+            "defaultTextIfEmpty": "Reporting is not yet installed. Follow <a href=\"https://simeoncloud.github.io/docs/#/reporting\">this guide</a> or complete these variables to install reporting.",
+            "category": "Integrations",
+            "isAdvanced": true,
+            "isSecret": false,
+            "readOnly": false
+        },
+        "LogAnalyticsResourceId": {
+            "type": "string",
+            "title": "Log Analytics Workspace Resource Id",
+            "description": "The log analytics workspace id used during Power BI report installation.",
+            "defaultTextIfEmpty": "Reporting is not yet installed. Follow <a href=\"https://simeoncloud.github.io/docs/#/reporting\">this guide</a> or complete these variables to install reporting.",
+            "category": "Integrations",
+            "isAdvanced": true,
+            "isSecret": false,
+            "readOnly": false
+        },
+        "SimeonSyncIntegrationAppId": {
+            "type": "string",
+            "title": "Simeon Sync Integration App Id",
+            "description": "App registration that can be used for all Simeon Cloud Integrations.",
+            "defaultTextIfEmpty": "Reporting is not yet installed. Follow <a href=\"https://simeoncloud.github.io/docs/#/reporting\">this guide</a> or complete these variables to install reporting.",
+            "category": "Integrations",
+            "isAdvanced": true,
+            "isSecret": false,
+            "readOnly": false
+        },
+        "SimeonSyncIntegrationAppSecret": {
+            "type": "string",
+            "title": "Simeon Sync Integration App Secret",
+            "description": "Client secret for shared integration app registration.",
+            "defaultTextIfEmpty": "Reporting is not yet installed. Follow <a href=\"https://simeoncloud.github.io/docs/#/reporting\">this guide</a> or complete these variables to install reporting.",
```

Comment:
why not use the default keyword?

Reply:
Because its not the default of the value, its more default description or hint text. Maybe better name would be "hintText"

Reply:
I don't agree 100% with the json schema's usage of the default key word, but we will stick to the standards of using default. I updated to default.

This is why I caved: 

The default keyword specifies a default value. This value is not used to fill in missing values during the validation process. Non-validation tools such as documentation generators or form generators may use this value to give hints to users about how to use a value. However, default is typically used to express that if a value is missing, then the value is semantically the same as if the value was present with the default value. The value of default should validate against the schema in which it resides, but that isn't required.